src/assets/styles/theme.scss

```diff
@@ -50,6 +50,7 @@ $BCgovAGold4: #ffd4a2;
 
 // Error Colors
 $BCgovInputError: #C62828; // aka "red darken-3"
+$BCgovError: #d3272c;
```

Comment:
The reason the other variables here are called "BCgovXXX" is because they came from an official BC Gov web style template.

Did this new variable come from that template as well? If not, please use a different name.

Reply:
Sure, will do

Reply:
Updated in next commit